subprojects/docs/src/docs/userguide/running-builds/configuration_cache.adoc

```diff
@@ -23,9 +30,9 @@ In other words, the configuration cache caches the output of the configuration p
 ====
 This feature is currently *experimental* and not enabled by default.
 
-Not all <<#core_plugins, core Gradle plugins>> are supported.
-Your build and the plugins you depend on might require changes to fulfil the <<#requirements, requirements>>.
-Some Gradle features are <<#not_yet_implemented, not yet implemented>>.
+Not all <<configuration_cache#core_plugins, core Gradle plugins>> are supported.
+Your build and the plugins you depend on might require changes to fulfil the <<configuration_cache#requirements, requirements>>.
+Some Gradle features are <<configuration_cache#not_yet_implemented, not yet implemented>>.
```

Comment:
I'd think about explicitly mentioning sync, saying that it is not improved by configuration caching yet but will be.